mono/mini/genmdesc.c

```diff
@@ -220,7 +220,7 @@ init_table (void) {
 
 static void
 output_char (FILE *f, char c) {
-	if (isalnum (c))
+	if (isalnum ((unsigned char)c)) // Cast to unsigned char to work around a bug in MSVCRT in debug builds
```

Comment:
Update: Each byte in the "spec" field does something different. Some values in the field are ASCII and some are not. The "euro symbol" was the "len" field and the value was 128, which is a valid value in this case and validly read from the file. I'll adjust the comment.

It does concern me genmdesc does almost no input validation.